cherab/tools/primitives/axisymmetric_mesh.pyx

```diff
@@ -1,24 +1,37 @@
+# Copyright 2016-2022 Euratom
+# Copyright 2016-2022 United Kingdom Atomic Energy Authority
+# Copyright 2016-2022 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
 
-import numpy as np
-cimport numpy as np
-from libc.math cimport cos, sin
 from raysect.primitive.mesh.mesh cimport Mesh
 
+from .toroidal_mesh import toroidal_mesh_from_polygon
 
-cdef double DEG2RAD = 2 * np.pi / 360
 
-
-cpdef Mesh axisymmetric_mesh_from_polygon(np.ndarray polygon, int num_toroidal_segments=500):
+cpdef Mesh axisymmetric_mesh_from_polygon(object polygon, int num_toroidal_segments=500):
     """
     Generates an Raysect Mesh primitive from the specified 2D polygon.
```

Comment:
The only reason to keep this function is for backwards compatibility. I think we can add a deprecation warning. Could you add the following text to the beginning of the docstring?
```
.. deprecated:: 1.4.0
       Use `toroidal_mesh_from_polygon` with `toroidal_extent=360` instead.
```